sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -470,6 +470,9 @@ def test_combining_with_accumulation_mode_and_fanout(self):
       ts.add_elements([i])
     ts.advance_watermark_to_infinity()
 
+    def is_early_firing(element, num_partitions):
+      return 0 if element < 15 else 1
```

Comment:
Wouldn't it be better to filter on the pane info itself?

Reply:
PaneInfo is not supported yet in Python.
https://issues.apache.org/jira/browse/BEAM-3759

Reply:
This was fixed last year. Looks like the bug was never closed.

Reply:
I tried to do this, but it doesn't work. Can't get the timing by calling pane.timing().

AttributeError: '_DoFnParam' object has no attribute 'timing'

def partition_firings(element, num_partitions, pane=beam.DoFn.PaneInfoParam):
      if pane.timing() == windowed_value.PaneInfoTiming.EARLY:
        return 0
      elif pane.timing() == windowed_value.PaneInfoTiming.ON_TIME:
        return 1
      else:
        return 2